### PR TITLE
docs: Include version number in docs commit and PR

### LINKED ISSUE #757: "Update docs for latest fastlane release" PRs should include version number in issue title
...

### PR DESCRIPTION
**Pull request by janpio**
closes https://github.com/fastlane/docs/issues/757

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -378,14 +378,15 @@ desc "Update the actions.md on https://docs.fastlane.tools"
...
+  version = options[:version]
 
   unless debug
     verify_env_variables
     ensure_actions_config_items_formatting
   end
 
   template_path = File.expand_path("./assets/render_plugin.md.erb")
-  clone_docs(debug: debug) do
+  clone_docs(debug: debug, version: version) do
```
Comment (joshdholtz) [inline fastlane/Fastfile]: This shouldn't be needed? `clone_docs` doesn't have `version` in its signature 🤔 
```suggestion
  clone_docs(debug: debug) do
```

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Love this change 👍 But one comment on `version` being passed into clone docs

File: fastlane/Fastfile (hunk 1 above)
Comment (janpio) [reply fastlane/Fastfile]: true, not sure where this is coming from.
brain fart.

**Review by joshdholtz — APPROVED** [review]
🔥

**Pull request merged**